CHANGELOG.md

```diff
@@ -5,6 +5,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+- Bumped ancla to v0.3.3 and changed server log source address field. [#231](https://github.com/xmidt-org/tr1d1um/pull/231)
+- Bumped ancla to v0.3.5, bumped argus to v0.6.0, and changed errorEncoder to pull logger from context. [#233](https://github.com/xmidt-org/tr1d1um/pull/233)
```

Comment:
From a release perspective, we don't need to know about intermediate increases in package versions.  I would restructure these lines like this:
```suggestion
- Bumped argus to v0.6.0. [#233](https://github.com/xmidt-org/tr1d1um/pull/233)
- Bumped ancla to v0.3.5:
  - Changed server log source address field. [#231](https://github.com/xmidt-org/tr1d1um/pull/231)
  - Changed errorEncoder to pull logger from context. [#233](https://github.com/xmidt-org/tr1d1um/pull/233)
```

Reply:
As a part of the previous release, we bumped ancla to v0.3.4, so I'm not sure about the first bullet? Was the change already a part of the last release?

Reply:
The change should have been a part of the last release. I bumped ancla in pull #231 and then you bumped it in pull #232 and made a release then. I believe I didn't handle the conflicts correctly when I pulled from main and that's why that first bullet is here now.